ci/prow/e2e-prebuilt-kernel-module

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+echo "Label the node so that it gets the module..."
+kubectl label node minikube wants-module=kmm_ci_a
+
+echo "Deploy KMMO..."
+make deploy
+
+echo "Wait until the KMMO Deployment is Available and show info if it fails..."
+timeout 1m kubectl wait --for condition=Available deployments.apps -n kmm-operator-system kmm-operator-controller-manager --request-timeout=1m 
```

Comment:
Why are we using both `timeout 1m` and `--request-timeout=1m`? I think we should use the same mechanism everywhere.